### DISCUSSION TITLE
Is it possible to influence Charts height?

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#2511] [Q&A] [answered]
Ok ok I know almost for certain that this is something that is going to be available in 0.9.4 but I have to ask (even though I promised to wait until that version)

But here goes: Can you **scale a chart down in height** and **keep its width**?

The reason I´m asking is because I don´t seem to be able to influence the Chart height even with styling like ` Style="height:50px"`. Its always the same height....

As you can see here the chart takes way to much height. I would like to have it 50-80% the height.
![image](https://user-images.githubusercontent.com/2386572/122936776-f2dfb380-d360-11eb-9162-c727d7daf7d9.png)

This is the simple code got ([see the example code](https://github.com/Megabit/Blazorise/blob/master/Demos/Blazorise.Demo/Pages/Tests/ChartsPage.razor))
```xml
<Row >
    <Column ColumnSize="ColumnSize.IsThird.OnWidescreen">
        <Card Margin="Margin.Is4.OnY">
            <CardHeader>
                <CardTitle>Container</CardTitle>
            </CardHeader>
            <CardBody>
                <Chart @ref="barChart" Type="ChartType.Bar" TItem="double"  />
            </CardBody>
        </Card>
    </Column>
</Row>
```

**Comment by stsrki** [comment, marked as answer]
I can't confirm now, not at home. But I think it is possible with aspect ratio option. I worked on that recently.

**Reply by sturlath** [reply]
Ahh see it! It worked! I didn´t spot the ChartOption option. When I set AspectRation to 6 it looks like I wanted. Thanks again!

**Reply by stsrki** [reply]
Cool. Glad I could help once again :)
